docs/model-repos.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Model Repos docs
+---
+
+<h1>Model Repos docs</h1>
+
+## What are model cards and why are they useful?
+
+The model cards are markdown files that accompany the models and provide very useful information. They are extremely important for discoverability, reproducibility and sharing! They are the `README.md` file in any repo.
+
+## When sharing a model, what should I add to my model card?
+
+The model card should describe:
+- the model
+- its intended uses & potential limitations, including bias and ethical considerations as detailed in [Mitchell, 2018](https://arxiv.org/abs/1810.03993)
+- the training params and experimental info (you can embed or link to an experiment tracking platform for reference)
+- which datasets did you train on and your eval results
+
+If needed you can find the specification [here](https://raw.githubusercontent.com/huggingface/huggingface_hub/main/modelcard.md).
+
+## How are model tags determined?
+
+The model cards have a YAML section that specify metadata. These are the fields
+
+```
+---
+language: "ISO 639-1 code for your language, or `multilingual`"
+thumbnail: "url to a thumbnail used in social sharing"
+tags:
+- array
+- of
+- tags
+license: "any valid license identifier"
+datasets:
+- array of dataset identifiers
```

Comment:
i find the "array" description a bit confusing (especially if you don't read the `tags` example above). how about doing something more explicit like the following?

```
tags:
- tag1
- tag2
license: "any valid license identifier"
datasets:
- dataset1
- dataset2
```

Reply:
Updated to something clearer as you suggested :) thanks!